solr/core/src/test/org/apache/solr/search/TestFilteredDocIdSet.java

```diff
@@ -127,22 +130,42 @@ public void testNullDocIdSet() throws Exception {
     IndexSearcher searcher = newSearcher(reader);
     Assert.assertEquals(1, searcher.search(new MatchAllDocsQuery(), 10).totalHits.value);
     
-    // Now search w/ a Filter which returns a null DocIdSet
-    Filter f = new Filter() {
+    // Now search w/ a Query which returns a null DocIdSet
```

Comment:
Query doesn't return DocIdSet.  You changed it to return a null Scorer which is close enough to the original intent.

Reply:
Thanks, will change that